examples/getting-started-movielens/02-ETL-with-NVTabular.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>this fixes https://github.com/NVIDIA-Merlin/Merlin/issues/163</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>